flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/WindowAggregateTest.scala

```diff
@@ -239,4 +252,179 @@ class WindowAggregateTest extends TableTestBase {
       )
     streamUtil.verifySql(sql, expected)
   }
+  
+   @Test
+  def testMaxAggregatation(): Unit = {
```

Comment:
Basically implement tests which are equivalent to the other tests in `WindowAggregateTest`. These tests do not execute a query but compare the plans generated by the optimizer.